apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingStatusService.kt

```diff
@@ -289,6 +295,22 @@ class BehandlingStatusServiceImpl(
         }
     }
 
+    private fun haandterFeilutbetaling(behandling: Behandling) {
+        val brevutfall = behandlingInfoDao.hentBrevutfall(behandling.id)
+        if (brevutfall?.feilutbetaling?.valg in listOf(FeilutbetalingValg.JA_VARSEL, FeilutbetalingValg.JA_INGEN_TK)) {
+            logger.info("Oppretter oppgave av type ${OppgaveType.TILBAKEKREVING} for behandling ${behandling.id}")
+            oppgaveService.opprettNyOppgaveMedSakOgReferanse(
+                referanse = behandling.id.toUUID30().value,
```

Comment:
Er det betre å utvide referanse-feltet til å takle ein heil UUID på 36 teikn, mon tru? Slår meg som noko vi kjem til å gløyme ein gong i framtida og dermed brenne oss på.

Reply:
@madsop-nav : Økonomi støtter bare 30 tegn i sitt felt for referanse til behandling. Når vi iverksetter en behandling, så er det behandlingsid som sendes over (forkortet til 30 tegn). Når vi så får tilbake et kravgrunnlag, så får vi denne id'en på 30 tegn tilbake.

Reply:
Skjønner! Da er det jo ikkje mykje anna enn det her vi får gjort 👍